docs/user_guide/index.md

```diff
@@ -61,27 +61,56 @@ module use /g/data/hh5/public/modules
 module load conda/analysis3-unstable
 ```
 
-Then simply launch `benchcab` from within your work directory:
+Currently, `benchcab` can only run CABLE for single sites. To run the single site tests, run
```

Comment:
```suggestion
Currently, `benchcab` can only run CABLE for flux sites. To run the flux site tests, run
```
Otherwise, it's confusing with "forty-two site" and "five site" tests.